### PR TITLE
Project Euler Problem 40 README and Python solution

### PR DESCRIPTION
**Pull request by adeen-s**
**Fixes issue:**
Adds the python solution of Problem 40 of Project Euler.

**Changes:**
Add new file problem_040.py which outputs the solution to Project Euler Problem 40

### CODE REVIEW

**Comment by adeen-s** [comment]
@danos7777 I am not sure what you are trying to say. Maybe you commented in the wrong PR

File: code/online_challenges/src/project_euler/problem_040/problem_040.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Made by adeen-s
+# Part of cosmos by OpenGenus
+
+nums = "" # The string containing the sequence of numbers
+i = 0 # Used for iteration
+
+while len(nums) < 1000002: # Loop to add the sequence of numbers to the list
+    nums = nums + str(i) 
+    i = i + 1
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_040/problem_040.py]: `+=` operator.

File: code/online_challenges/src/project_euler/problem_040/problem_040.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+answer = 1 # Initialising answer which will store final computed answer
+i = 1 # Reset value for use in next loop
+
+while i <= 1000000: # Loop to generate the answer
+    answer = answer * int(nums[i])
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_040/problem_040.py]: `*=` operator.

File: code/online_challenges/src/project_euler/problem_040/problem_040.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Made by adeen-s
+# Part of cosmos by OpenGenus
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_040/problem_040.py]: Not needed.

File: code/online_challenges/src/project_euler/problem_040/problem_040.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+while len(nums) < 1000002: # Loop to add the sequence of numbers to the list
+    nums = nums + str(i) 
+    i = i + 1
+
+answer = 1 # Initialising answer which will store final computed answer
+i = 1 # Reset value for use in next loop
+
+while i <= 1000000: # Loop to generate the answer
+    answer = answer * int(nums[i])
+    i = i * 10
```
Comment (arnavb) [inline code/online_challenges/src/project_euler/problem_040/problem_040.py]: Same here.

File: code/online_challenges/src/project_euler/problem_040/problem_040.py (hunk 3 above)
Comment (adeen-s) [reply code/online_challenges/src/project_euler/problem_040/problem_040.py]: Well that's just an old habit from the hacktoberfest days. Fixed it all now.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**